src/main/java/frc/robot/subsystems/superstructure/arm/ArmIOReal.java

```diff
@@ -21,10 +23,15 @@ public class ArmIOReal implements ArmIO {
     private final TalonFX leftPivotMotor;
     private final TalonFX rightPivotMotor;
 
-    private final MotionMagicExpoTorqueCurrentFOC motionMagicExpoTorqueCurrentFOC;
+//    private final MotionMagicExpoTorqueCurrentFOC motionMagicExpoTorqueCurrentFOC;
```

Comment:
You can get rid of this since sysID arm seems to only work for voltage. And I don't have a problem with using voltage for it since it works so well already.

Reply:
I'm willing to try `TorqueCurrentFOC` again, as I'm not confident we did it correctly or that we got good data - not worth it?